src/NuGet.Clients/NuGet.PackageManagement.UI/Telemetry/UINuGetActionSource.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Tracks where a NuGet action (install, update or uninstall) over one or more packages were triggered in PM UI
+    /// </summary>
+    public enum UINuGetActionSource
```

Comment:
How about `NuGetUIActionSource` instead?

Reply:
Yeah, probably better, matches things like `UIActionEngine`. Potentially `PMUIActionSource`.

Reviewing it the first time without the docs it just wasn't clear enough.